packages/select/src/components/query-list/queryList.tsx

```diff
@@ -189,6 +182,19 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         }
     }
 
+    // TODO resetActiveItem = this.props.resetOnQuery
```

Comment:
do you have plans to do this before merge or just leaving this in there?

Reply:
it's a followup, separate feature. not before merging.